packages/client/src/App.tsx

```diff
@@ -307,11 +384,9 @@ export function App(props: IAppProps) {
                                             />
                                             <ProtectedRoute
                                               exact
-                                              path={routes.VERIFY_COLLECTOR}
-                                              component={VerifyCollector}
-                                            />
```

Comment:
We don't want to remove this route now do we?

Reply:
Thanks, good catch